src/System.Console/src/System.Console.csproj

```diff
@@ -11,14 +11,16 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.3</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(PackageTargetFramework)'==''">netstandard1.3</PackageTargetFramework>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
```

Comment:
If so, there are several places this needs to be fixed e.g. [System.IO.FileSystem](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/src/System.Runtime.csproj) , [System.Runtime](https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/src/System.IO.FileSystem.csproj)